Meta/.shell_include.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
This file is for sourcing, not executing, right?

* Why does it have the x bit set?
* Why does it have a run line?
* Why does its name start with a `.`? (I'm guessing that's so that it doesn't show up in tab completion, but if it didn't have +x set that wouldn't be a problem either.)

I'm asking because I ran `% rg MKE2FS_PATH` earlier today to find where that var is set, and `rg` apparently doesn't look in hidden files by default. (This is arguably weird of `rg`!)

Reply:
I basically just copied what `.ports_include.sh` did, without much thought for broken grep implementations. `shellcheck` also is unhappy if I don't add a shebang.

Reply:
Is `shellcheck` also unhappy if the file isn't marked as executable?

Reply:
Yes, it is.

Reply:
Made #16588 to change this script as I suggested. I suppose CI lint will tell me what shellcheck thinks :)